services/ingestion/block_tracking_subscriber.go

```diff
@@ -219,13 +221,22 @@ func (r *RPCBlockTrackingSubscriber) subscribe(ctx context.Context, height uint6
 				for _, evt := range blockEvents.Events {
 					r.keyLock.NotifyTransaction(evt.TransactionID)
 				}
-				r.keyLock.NotifyBlock(
-					flow.BlockHeader{
-						ID:     blockEvents.BlockID,
-						Height: blockEvents.Height,
-					},
-				)
-				lastReceivedHeight = blockHeader.Height
```

Comment:
I think we still need this to support reconnects

Reply:
Oh, good catch, that was accidental :sweat_smile:

Reply:
Brought it back. Nice catch :100: